packages/rpc-core/src/rpc-methods/__tests__/get-genesis-hash-test.ts

```diff
@@ -0,0 +1,41 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+
+import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+// TODO: We may want to replace the higher clusters with just the local
+// validator, since this is within the live test suite anyhow.
+// Also, with further manipulation capabilities over the local validator,
+// we can explore error testing as well
+const genesisHashMap: [string, Base58EncodedAddress][] = [
+    ['https://api.devnet.solana.com', 'EtWTRABZaYq6iMfeYKouRu166VU2xqa1wcaWoxPkrZBG' as Base58EncodedAddress],
+    ['https://api.testnet.solana.com', '4uhcVJyU9pJkvQyS88uRDiswHXSCkY3zQawwpjk2NsNY' as Base58EncodedAddress],
+    ['https://api.mainnet-beta.solana.com', '5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d' as Base58EncodedAddress],
+];
```

Comment:
I'm a bit hesitant about hitting live RPC nodes in CI tests, but at the same time I don't think we can test this with the current local validator. I think I'd lean toward just having a `todo` test for now.

Reply:
Let me see if this one idea works, if not I'll make it a `todo`

Reply:
That should do it!